vite-svg-loader.js

```diff
@@ -51,7 +51,9 @@ module.exports = function svgLoader (options = {}) {
         transformAssetUrls: false
       })
 
-      return `${code}\nexport default { render: render }`
+      const template = code instanceof Promise ? await code : code;
```

Comment:
Not sure, why this is happening, but in Vite dev mode the SVGs could not be loaded, because `code` returned an unresolved promise. The build mode was running fine though.

Reply:
Out of curiosity, why does this file need to change at all?

Reply:
Because the Vite dev server wasn't able to load any SVGs, because the vite-svg-loader somehow didn't return the code as a string, but as a promise. Can you confirm that, or do SVGs load when runing `npm run dev` in the Statamic repo? It has nothing to do with the PR though, I just noticed that.

Reply:
They're returned intentionally as a promise, rather than a string because of the way they get imported. I've never ran into an issues with it. 🤔

Reply:
![Screenshot 2024-08-26 at 16 06 02](https://github.com/user-attachments/assets/30fec43c-a41a-4701-8ec1-753e625c9f71)

That's interesting. This is the error I get and the template generated by the `vite-svg-loader` looks like this:

```
[object Promise]
export default { render: render }
```